docs/hw/memory/g80-surface.rst

```diff
@@ -122,93 +122,93 @@ The attributes defining a tiled surface are:
 - depth: surface depth [size in z dimension] in elements
 
 .. todo:: check bounduaries on them all, check tiling on GF100.
-.. todo:: PCOPY surfaces with weird tile size
+.. todo:: PCOPY surfaces with weird block size
```

Comment:
technically, I think they had weird gob size, it's been ages since I looked at it though